paper.bib

```diff
@@ -1,5 +1,5 @@
 @article{boltzmann1868studien,
-  title={Studien über das {G}leichgewicht der lebendigen {K}raft zwischen bewegten materiellen {P}unkten},
+  title={Studien über das Gleichgewicht der lebendigen Kraft zwischen bewegten materiellen Punkten},
```

Comment:
Just a comment on this: In German, these words cannot be spelled with a lower case initial (regardless of their position in a sentence, sentence case vs. title case, etc.), so I think they need to be escaped with `{…}` here.

Reply:
ok, I'll unchange them. I didn't see them as proper names/places, but I don't know German rules

Reply:
Yes, this is a particularity of German grammar – *all* nouns (not just names, places, etc.) are capitalized.